apps/condo/domains/billing/schema/B2BAppIntegration.test.js

```diff
@@ -0,0 +1,174 @@
+const {
+    expectToThrowAccessDeniedErrorToResult, makeClient,
+} = require('@open-condo/keystone/test.utils')
+
+const { registerBillingReceiptsByTestClient } = require('@condo/domains/billing/utils/testSchema')
+const { createTestBillingIntegrationOrganizationContext } = require('@condo/domains/billing/utils/testSchema')
+const {
+    TestUtils,
+    OrganizationTestMixin,
+    BillingTestMixin,
+    B2BAppTestMixin,
+} = require('@condo/domains/billing/utils/testSchema/testUtils')
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/miniapp/constants')
+const { createTestB2BAppContext } = require('@condo/domains/miniapp/utils/testSchema')
+
+const { registerBillingReceiptFileByTestClient } = require('../utils/testSchema')
+
+describe('B2BAccess check with B2BAccessToken', () => {
+
+    let environment1 = new TestUtils([BillingTestMixin, B2BAppTestMixin])
+    let environment2 = new TestUtils([OrganizationTestMixin])
```

Comment:
Why not give meaningful names

Reply:
but it is an environment 
it's an object with organization, employees, Properties and so on